### PR TITLE
workflows: workflow representation fix

### PR DESCRIPTION
**Pull request by GiorgosPa**
- Calculates the path of tasks that workflow has
  followed and represents with different color
  the completed, last and blocked tasks.

Signed-off-by: Georgios Papoutsakis georgios.papoutsakis@cern.ch

### CODE REVIEW

File: invenio/modules/workflows/static/css/workflows/workflows.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+button[status=completed]{
```
Comment (jalavik) [inline invenio/modules/workflows/static/css/workflows/workflows.css:20]: Need to add license info. See how any other custom CSS in pu has done it.

File: invenio/modules/workflows/views/holdingpen.py [hunk 2]
```diff
@@ -82,23 +83,26 @@ def index():
...
+            if ObjectVersion.MAPPING[int(request.args.get('version'))] not in my_tags:
+                my_tags += ObjectVersion.MAPPING[[int(request.args.get('version'))]]
+        tags_to_print = ""
+        for tag in my_tags:
+            if tag:
+                tags_to_print += tag + ','
+        return dict(bwolist=bwolist,
+                    action_list=action_list,
+                    tags=tags_to_print)
+    except Exception as e:
```
Comment (jalavik) [inline invenio/modules/workflows/views/holdingpen.py]: You can revert this

File: invenio/modules/workflows/views/holdingpen.py [hunk 3]
```diff
@@ -164,7 +199,68 @@ def details(objectid):
                            data_preview=formatted_data,
                            workflow_func=extracted_data['workflow_func'],
                            workflow=extracted_data['w_metadata'],
-                           task_results=results)
+                           task_results=results,
+                           task_history=task_history)
+
+
+def is_sublist(list_a, list_b):
```
Comment (jalavik) [inline invenio/modules/workflows/views/holdingpen.py]: So you can move the non routed function into `utils.py` or something.

**Comment by GiorgosPa** [comment]
Done.

**Comment by GiorgosPa** [comment]
Rebased.

**Pull request closed without merging**